### ISSUE TITLE
[a11y] Cannot close GeoLocate window in WB

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1496]
![msedge_Yg09Px0D4L](https://user-images.githubusercontent.com/37256050/167680558-ab4eda38-7491-4acf-af23-ba1d59799677.gif)

db: `oldfish`
user: `abentley`
